### PR TITLE
Add superfluid flow test

### PR DESCRIPTION
**Pull request by mattverse**
Closes: #XXX

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

### CODE REVIEW

**Comment by mattverse** [comment]
Note to reviewers: this test is mostly integrations of tests for different components

File: x/superfluid/keeper/superfluid_test.go [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+				Identifier:            "day",
+				StartTime:             suite.ctx.BlockTime(),
+				Duration:              time.Hour * 24,
+				CurrentEpoch:          0,
+				CurrentEpochStartTime: time.Time{},
+				EpochCountingStarted:  false,
+			}
+			suite.app.EpochsKeeper.SetEpochInfo(suite.ctx, epochInfo)
+
+			epochs.BeginBlocker(suite.ctx, *suite.app.EpochsKeeper)
```
Comment (ValarDragon) [inline x/superfluid/keeper/superfluid_test.go:56]: Would be ideal if we can switch to App.BeginBlock() / app.EndBlock(), so we can understand whats the pipeline of all the moving components

Comment (ValarDragon) [reply x/superfluid/keeper/superfluid_test.go:56]: (Somewhat worried that the order of calls in here will be different than what we actually use)

Comment (mattverse) [reply x/superfluid/keeper/superfluid_test.go:56]: good catch!

**Comment by mattverse** [comment]
Updated test so that it uses app.BeginBlock, app.EndBlock, app.Commit

**Comment by mattverse** [comment]
Closing in favor of https://github.com/osmosis-labs/osmosis/pull/1070

**Pull request closed without merging**